src-vue/src/views/ModsView.vue

```diff
@@ -44,6 +45,39 @@ export default defineComponent({
                     position: 'bottom-right'
                 });
             });
+    },
+    methods: {
+        async updateWhichModsEnabled(mod: NorthstarMod) {
+            this.global_load_indicator = true;
+
+            // Setup up struct
+            let game_install = {
+                game_path: this.$store.state.game_path,
+                install_type: this.$store.state.install_type
+            } as GameInstall;
+
+            // enable/disable specific mod
+            try {
+                await invoke("set_mod_enabled_status_caller", {
+                    gameInstall: game_install,
+                    modName: mod.name,
+                    isEnabled: !mod.enabled, // TODO this might set it to the old state
```

Comment:
Still TODO? :)

Reply:
Whoops, fixed in ab50b171f735ad8cebc390ec84f8da6aa8208dfb